tests/data/tagged_codelist/tag_sector.yaml

```diff
@@ -6,4 +6,3 @@
       weight: Energy
```

Comment:
Are the the "value"-entries actually used anywhere?

Reply:
What do you mean by the "value" entries?

Reply:
There are two attributes "value_bool" and "value_number" just above the commented line...

Reply:
Ah right, sorry didn't see those. I don't think they're used, I'll double check. Should I remove them if they're not used?

Reply:
Removed it, didn't break the tests on my machine. If they also run through here I'll go ahead with the merge.